src/diffusers/schedulers/__init__.py

```diff
@@ -38,6 +38,8 @@
     _dummy_modules.update(get_objects_from_module(dummy_pt_objects))
 
 else:
+    _import_structure["deprecated.scheduling_karras_ve"] = ["KarrasVeScheduler"]
```

Comment:
```suggestion
    _import_structure["deprecated"] = ["scheduling_karras_ve", "KarrasVeScheduler"]
```

Reply:
@a-r-r-o-w 
can we apply the changes @patrickvonplaten suggested here?